tests/vmi_cloudinit_test.go

```diff
@@ -530,15 +530,16 @@ func verifyUserDataVMI(vmi *v1.VirtualMachineInstance, commands []expect.Batcher
 	Expect(console.SafeExpectBatch(vmi, commands, int(timeout.Seconds()))).To(Succeed())
 }
 
-func checkCloudInitFile(vmi *v1.VirtualMachineInstance, testFile, testData string) {
+func checkCloudInitFile(vmi *v1.VirtualMachineInstance, testFile, testData string) string {
```

Comment:
Following this change there are now two assertions asserting the file's content:
```
&expect.BExp{R: testData},
``
and
```
Expect(initfile).To(ContainSubstring(string(buf)))
```

Although the first one is implicit, please consider dropping the second one, as it is redundant.
Alternatively, we could have a function that returns a file's content without setting an expectation and keeping the explicit assertion.